app/src/main/res/layout-land/profile_progress_header.xml

```diff
@@ -46,12 +46,16 @@
       android:id="@+id/profile_name_text_view"
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
+      android:layout_marginStart="44dp"
+      android:paddingEnd="185dp"
       android:layout_marginTop="12dp"
       android:fontFamily="sans-serif-medium"
       android:text="@{viewModel.profile.name}"
       android:textColor="@color/oppiaPrimaryText"
       android:textSize="28sp"
-      app:layout_constraintStart_toStartOf="@+id/ongoing_topics_container"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintHorizontal_bias="0.0"
+      app:layout_constraintStart_toEndOf="@+id/profile_edit_image"
```

Comment:
Please don't introduce random margin and padding. In oppia every size of every item is dependent upon mocks and we try to make it match with that as much as possible.

To fix UI for this screen you can use below code:
```
 <TextView
      android:id="@+id/profile_name_text_view"
      android:layout_width="0dp"
      android:layout_height="wrap_content"
      android:layout_marginTop="12dp"
      android:fontFamily="sans-serif-medium"
      android:text="@{viewModel.profile.name}"
      android:textColor="@color/oppiaPrimaryText"
      android:textSize="28sp"
      app:layout_constraintEnd_toEndOf="@id/completed_stories_container"
      app:layout_constraintStart_toStartOf="@+id/ongoing_topics_container"
      app:layout_constraintTop_toTopOf="parent" />
```

Reply:
Thanks for the help!

Reply:
@khushal1707 Please do not resolve comments. The person who has started the comment thread should resolve it.